src/Symfony/Component/Validator/Constraints/Expression.php

```diff
@@ -30,6 +30,7 @@ class Expression extends Constraint
 
     public $message = 'This value is not valid.';
     public $expression;
+    public $values = array();
```

Comment:
should it be named $variables?

Reply:
I've named it according to second argument in ExpressionLanguage::evaluate. If $variables is better name, maybe it's worth it to change it there too